circle.yml

```diff
@@ -0,0 +1,26 @@
+machine:
+  java:
+    version: oraclejdk8
+  environment:
+    QEMU_AUDIO_DRV: none
+    ANDROID_SDK: "android-25"
+    BUILD_TOOLS: "25.0.3"
+    ANDROID_EMULATOR: "circleci-android24"
+    ANDROID_ABI: "armeabi-v7a"
+
+dependencies:
+  pre:
+    # Android SDK Platform
+    - if [ ! -d "/usr/local/android-sdk-linux/platforms/$ANDROID_SDK" ]; then echo y | android update sdk --no-ui --all --filter "$ANDROID_SDK"; fi
+    # Android SDK Build-tools
+    - if [ ! -d "/usr/local/android-sdk-linux/build-tools/$BUILD_TOOLS" ]; then echo y | android update sdk --no-ui --all --filter "build-tools-$BUILD_TOOLS"; fi
+    # Android m2repository and extra support
+    - echo y | android update sdk --no-ui --all --filter "extra-google-m2repository, extra-android-m2repository, extra-android-support"
+
+  cache_directories:
+    - /usr/local/android-sdk-linux
+    - ~/.gradle
+
+test:
+  override:
+    - ./gradlew build build
```

Comment:
@jcesarmobile `connectedCheck` when you need to run a test on the emulator. `build` when you only need to check if it build without error. Going to replace the `build build` by `clean build` :D